app/Messaging/queues.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use Vinelab\Bowler\Facades\Registrator;
+
+Registrator::queue('instagram_insights', 'App\Messaging\Handlers\InstagramInsightsHandler', [
+    'exchangeName' => 'instagram_insights',
```

Comment:
only add the options you want to change, otherwise remove to take default from Bowler.